bin/postman-test/collections/system-management.postman_collection.json

```diff
@@ -132,7 +128,7 @@
           "raw": ""
         },
         "url": {
-          "raw": "{{systemManagementUrl}}/api/v1/health/device-virtual,edgex-core-command,edgex-core-data,edgex-core-metadata,edgex-export-client,edgex-export-distro,edgex-support-logging,edgex-support-notifications,edgex-support-scheduler",
+          "raw": "{{systemManagementUrl}}/api/v1/health/edgex-core-command,edgex-core-data,edgex-core-metadata,edgex-export-client,edgex-export-distro,edgex-support-logging,edgex-support-notifications",
```

Comment:
Why was edgex-support-scheduler removed from here (and elsewhere)?

Reply:
This was inadvertent, and has been addressed.